src/TrueLayer/Payments/Model/Provider.cs

```diff
@@ -23,5 +23,39 @@ public record UserSelected : IDiscriminated
             /// </summary>
             public ProviderFilter? Filter { get; init; }
         }
+
+        /// <summary>
+        /// Represents provider options that indicates that the provider for this payment is preselected
+        /// </summary>
+        [JsonDiscriminator("preselected")]
+        public record Preselected : IDiscriminated
+        {
+            public Preselected(string providerId, string schemeId, RemitterAccount remitter)
+            {
+                ProviderId = providerId.NotNull(nameof(providerId));
+                SchemeId = schemeId.NotNull(nameof(schemeId));
+                Remitter = remitter.NotNull(nameof(remitter));
+            }
+
+            /// <summary>
+            /// Gets the provider type
+            /// </summary>
+            public string Type => "preselected";
+
+            /// <summary>
+            /// Gets the provider Id the PSU will use for this payment
+            /// </summary>
+            public string ProviderId { get; }
+
+            /// <summary>
+            /// Gets the id of the scheme to make the payment over
+            /// </summary>
+            public string SchemeId { get; }
+
+            /// <summary>
+            /// Gets the account details for the remitter
+            /// </summary>
+            public RemitterAccount Remitter { get; }
```

Comment:
As per the spec the remitter field should be optional here

Reply:
@tl-wajid-malik thank you. I have addressed it in da01b5d